eslint.config.mjs

```diff
@@ -63,6 +63,7 @@ export default defineConfig([
                     style: 'camelCase',
                 },
             ],
+            '@angular-eslint/prefer-signals': 'warn',
```

Comment:
unrelated to this line: is there a reason you did not use the inject here?

![alt text](image.png)

Reply:
These are no components, thus there is no dependency injection.